pkg/controllers/allocation/controller.go

```diff
@@ -66,13 +84,15 @@ func NewController(kubeClient client.Client, coreV1Client corev1.CoreV1Interface
 // Reconcile executes an allocation control loop for the resource
 func (c *Controller) Reconcile(ctx context.Context, object client.Object) (reconcile.Result, error) {
 	provisioner := object.(*v1alpha2.Provisioner)
+	c.batcher.Complete(fmt.Sprintf("%s/%s", provisioner.Name, provisioner.Namespace))
```

Comment:
Ahh I had half written a comment to rename this to Complete and then thinking further on it, I deleted the comment because I liked the explicit nature of the "Wait()" semantic. Sorry for the thrash!

Reply:
Similarly to the above comment on runtime.Object, batcher.Wait(provisioner)

Reply:
ha no worries, I'm not too attached to the names, so I'll change back. The inspiration was really WaitGroups